Gemfile

```diff
@@ -33,6 +33,7 @@ group :test, :development do
   gem 'awesome_print'
   gem 'dotenv-rails'
   gem 'pry'
+  gem 'faker'
```

Comment:
faker introduces randomness, and hence unreliability, into tests. instead use factory girl sequences